site/source/device-guides/mac/tor-mac.rst

```diff
@@ -11,14 +11,10 @@ Install Homebrew
 
     .. code-block:: bash
 
-        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 
 #. You will be prompted for your system password before installation; proceed with entering your password.  You may be asked more than once.
 
-    .. figure:: /_static/images/tor/install_homebrew.png
```

Comment:
This screenshot no longer represents the step it's supposed to illustrate.  It gets the chop.